src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
+Spring.demo.value= test
```

Comment:
The property key 'Spring.demo.value' should be in lowercase. Please change it to 'spring.demo.value'.